modules/serverless-kafka-sasl.adoc

```diff
@@ -0,0 +1,83 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-sasl-authentication-against-an-apache-kafka_{context}"]
+== Configuring SASL authentication
+
+.Prerequisites
+
+* A username and password for the Kafka cluster.
+* Choose the SASL mechanism to use, for example `PLAIN`, `SCRAM-SHA-256` or `SCRAM-SHA-512`.
+* If TLS is enabled, you also need the `ca.crt` certificate file for the Kafka cluster.
+
+Red Hat recommends to enable TLS in addition to SASL.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+----
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
```

Comment:
s/`<kafka-auth-secret>`/`<kafka_auth_secret>`/ 
Underscores for replaceable values.